spec/system/admin_notices_spec.rb

```diff
@@ -1,25 +1,43 @@
 # frozen_string_literal: true
 
 describe "Admin Notices", type: :system do
-  fab!(:admin)
-
   let(:admin_dashboard) { PageObjects::Pages::AdminDashboard.new }
 
   before do
     Fabricate(:admin_notice)
 
     I18n.backend.store_translations(:en, dashboard: { problem: { test_notice: "Houston" } })
+  end
+
+  context "when signed in as admin" do
+    fab!(:admin)
+
+    before { sign_in(admin) }
+
+    it "supports dismissing admin notices" do
+      admin_dashboard.visit
+
+      expect(admin_dashboard).to have_admin_notice(I18n.t("dashboard.problem.test_notice"))
```

Comment:
theres opportunity for better page object here, I see we use this multiple times

Reply:
Hm. What's the idea here? In two different tests I want to assert that the page has a certain notice.

The only thing I can see is hiding away the `I18n` part in the page object, but maybe that needs to be adopted as a broader convention, then?

Reply:
nah sorry we are good, I was on mobile and missread the line, all good 👍